Base.rte/Effects/Casings.ini

```diff
@@ -174,4 +174,24 @@ AddAmmo = AEmitter
 	AngularVel = 3
 	EmissionOffset = Vector
 		X = -2
-		Y = 0
\ No newline at end of file
+		Y = 0
+
+
+AddAmmo = MOSParticle
+	PresetName = Frag Grenade Pin
+	Mass = 0.05
+	HitsMOs = 0
+	GetsHitByMOs = 0
+	RestThreshold = 1000
+	SpriteFile = ContentFile
+		FilePath = Base.rte/Devices/Explosives/FragGrenade/FragGrenadePin.png
```

Comment:
Probably belongs in `Base.rte\Effects\Casings` since it's here and a shared "effect" particle and all that.
Also should probably just be `Grenade Pin`.